about.html

```diff
@@ -0,0 +1,157 @@
+          <div>
+            <h1 id="hero-top"><span>"Life is better at mentorship!"</span><br>BRIDGE GAP SUMMIT 2023</h1>
+          </div>
+          <div class="about-text">
+            <p class="text-about">
+              Potential is equally distributed; opportunity is not. MENTOR aims to drive equity 
+              and close the mentoring gap through quality mentoring relationships for young people.
+              We activate a diverse cross-sector movement that prioritizes 
+              relationships 
+            </p>
+          </div>
+          <div class="contact-about">
+            <p>
+              Please contact us per Email for nay further questions about Bridge gap mentorship.
+            </p>
+            <p class="email">bridgegap@mentoring.org</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section class="logo-info">
+        <p class="creative-logo">Bridge the Gap mentorship logo</p>
+        <p class="title-underline"></p>
+
+        <p class="logocc">
+          The logo of Bridge the Gap mentorship was decided through the logo
+          competition from 8, nov to 7, dec.
+        </p>
+        <div class="aboutlg">
+          <img class="abtLang" src="./assets/images/about_logo 1.png" />
+        </div>
+    </section>
+    <section class="previous-summit">
+        <div class="previous-summ-theme">
+          <p>see past mentorship programs</p>
+        </div>
+        <div class="summ-theme-text">
+            <p>Take a look at the past two mentorships events that took place in Nairobi</p>
+        </div>
+        <div class="previous-img">
+          <div class="aboutImage">
+            <img class="im1" src="./assets/images/about_2011 1.png" alt="image1">
+            <p class="img1">mentors summit 2022</p>
+          </div>
+            <div class="aboutImage"><img class="im1" src="./assets/images/about_2011 1.png" alt="image2"></div>
+            <p class="img2">mentors summit 2020</p>
+          </div>
+      </section>
+      <section>
+          <div class="partners">
+              <div class="partners-content">
+                  <div class="partners-title">
+                      <div class="partners-title-text">
+                          <p>Partner</p>
+                      </div>
+                      <div class="title-underline"></div>
+                  </div>
+                  <div class="group1Logo">
+                        <div class="partners-logo">
+                            <img src="./assets/images/mozilla 1.png" alt="mozilla">
+                            <img src="./assets/images/googlekorea 1.png" alt="googlekorea">
+                            <img src="./assets/images/naver 1.png" alt="naver">
+                        </div>
+                      <div class="partners-logo">
+                          <img src="./assets/images/daumkakaologo 1.png" alt="dau">
+                          <img src="./assets/images/airbnb 1.png" alt="airbnb">
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly make sure your partner's images have some `orange` background in that way you will stick to the original design as requested.
`Hint: You may consider achieving it like this:`
```html
<div class='partner__1'></div>
<div class='partner__2'></div>
```
```css
.partner__1 {
background-color: #6DB3F2;
background-image: url('images/checked.png');
/* OR */
background: url('images/checked.png'), #6DB3F2;
}
```

Reply:
Thank you for this. Allow me to fix this later